lib/statix/packet.ex

```diff
@@ -29,4 +32,12 @@ defmodule Statix.Packet do
   for {name, type} <- metrics do
     defp metric_type(unquote(name)), do: unquote(type)
   end
+  defp set_option(stats, _, nil), do: stats
+  defp set_option(stats, :sample_rate,sample_rate) do
+    [stats| ["|@", :io_lib.format('~.2f', [sample_rate])]]
+  end
+  defp set_option(stats, :tags, tags) do
+    [stats| ["|#", Enum.join(tags,",") ]]
```

Comment:
I think we need to escape `,` in tags, otherwise it will be interpreted incorrectly.